### DISCUSSION TITLE
Pages not deploying

### DISCUSSION THREAD

**Discussion by anjor** [community/community#128716] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

I tried deploying a website using github pages for the first time yesterday. Here's the repo: https://github.com/anjor/platypus-website/ .

The github action as recommended by material for mkdocs seems to be working fine, but the website isn't getting deployed.

I used to have a `anjor.github.io` repo with an `index.html` that I deleted yesterday -- I wonder if that has messed things up and is preventing me from deploying this website. 

Any ideas?

**Comment by arudeep15** [comment]
response #2

**Comment by anjor** [comment]
The problem was the settings weren't set up correctly. I was using the action provided by material for mkdocs but hadn't updated the source branch to gh-pages.
